packages/std-contracts/package.json

```
@@ -10,8 +10,9 @@
     "directory": "packages/std-contracts"
   },
   "scripts": {
+    "clean": "rimraf out && rimraf abi && rimraf types && rimraf cache",
```

same as in `solecs`, `abi` and `types` are cleared when calling `dist` and `types`, so we might not need them here and instead can call this `clean:forge` too?

```suggestion
    "clean:forge": "forge clean",
```